toolchains/docker/toolchain.bzl

```diff
@@ -38,21 +40,35 @@ docker_toolchain = rule(
         "tool_path": attr.string(
             doc = "Path to the docker binary",
         ),
+        "client_config": attr.string(
+            default = "",
+            doc = "A custom directory for the docker client config.json",
```

Comment:
The directory containing the docker client config.json. If not specified ...

Reply:
Done